docs/building-images.md

```diff
@@ -75,6 +75,7 @@ operatingSystem:
   kernelArgs:
   - arg1
   - arg2
+  fips: true
```

Comment:
This looks a bit odd to me, but not awful. @atanasdinov what do you prefer, `fips: true` or `enableFips: true`? I think I like the latter, but it's not a hill I'm gonna die on.

Reply:
I don't care either way, I've updated so that it's `enableFips`, as it does look a bit cleaner that way.